src/column.ts

```diff
@@ -120,21 +120,27 @@ export class ColumnFactory {
 
 export class RawColumn {

```

Comment:
There should be some unit tests added for this class to cover the new functionality.